executor/executor_required_rows_test.go

```diff
@@ -828,6 +828,10 @@ func (mp *mockPlan) GetExecutor() Executor {
 	return mp.exec
 }
 
+func (mp *mockPlan) Schema() *expression.Schema {
```

Comment:
Is this change necessary?

Reply:
https://github.com/pingcap/tidb/blob/b7894dfefbe4a435ca709597eadd72fb7a774f6a/executor/executor_required_rows_test.go#L808

here using `mockPlan` as children of a `MergeJoinPlan`, thus when building need to implement `*mockPlan.Schema()`